src/commands/run.js

```diff
@@ -9,21 +8,25 @@ module.exports = {
 	handler: argv => {
 		const { NODE_ENV } = process.env;
 		const envString = NODE_ENV || 'development';
-		console.log(chalk.green(`NODE_ENV=${envString}`));
+
+		console.log(chalk.gray('$ mope run'));
```

Comment:
curious why this is logged? I think yarn and make will log the same thing when it's called through an alias, and if it's not called through an alias, then I'm not sure why you would need it logged?

Reply:
yeah, so oddly I wasn't able to figure this out, but yarn doesn't log that. Here's the current output of running `yarn start:full`:

```
sadaf: ~/git_repo/mup-web [feature/webpack4] $ yarn run start:full
yarn run v1.3.2
$ make build-vendor && make build-trn && mope run
yarn clean:vendor
$ rm -rf build/vendor/
yarn build:vendor
$ mope build vendor
building vendor bundles...


built vendor bundles: react, vendor
yarn build:trnModules
$ mope build trn
Cleaning TRN modules directory
Transpiling TRN source to JSON...
Wrote TRN modules for 201 components
$ mope run
NODE_ENV=development
>> start-dev: Preparing the Dev App Server using existing vendor bundle...
(node:66562) DeprecationWarning: Tapable.plugin is deprecated. Use new API on `.hooks` instead
>> serverApp bundle built
>> browserApp bundle built
```

The `$ mope run` is my chalk log, otherwise yarn doesn't print it the same as for `$ mope build trn` and the rest. I added it in because it was helpful during testing but also not necessary and I can remove.

Reply:
technically not necessary because it's part of the output when you first run `yarn run start:full`:

```
$ make build-vendor && make build-trn && mope run
```

You get additional command logging from `make build-vendor` and `make build-trn` because `make` calls `yarn` again

I would leave out this log, or change it to something like `Starting dev environment...` or something even more specific to what the command does.

Reply:
I'll remove! There's plenty of other logging already.